Patch-Source: https://github.com/AyatanaIndicators/ayatana-indicator-keyboard/commit/191d5aeac6e44525f49db010d0613e40041750d0

From 191d5aeac6e44525f49db010d0613e40041750d0 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 9 Jan 2022 19:13:00 +0100
Subject: [PATCH] Try to dlopen .so.0 instead of .so

Some distributions do not provide a .so symlink in the default package,
and in case there are major breaking changes (.so.1) then the code
likely needs to get adjusted anyways.
---
 src/service.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/service.c b/src/service.c
index 6d1452da..73f61065 100644
--- a/src/service.c
+++ b/src/service.c
@@ -409,11 +409,11 @@ static void onDispose(GObject *pObject)
 
 static void indicator_keyboard_service_init(IndicatorKeyboardService *self)
 {
-    gchar *sLib = "libayatana-keyboard-x11.so";
+    gchar *sLib = "libayatana-keyboard-x11.so.0";
 
     if (ayatana_common_utils_is_lomiri())
     {
-        sLib = "libayatana-keyboard-lomiri.so";
+        sLib = "libayatana-keyboard-lomiri.so.0";
     }
 
     m_pLibHandle = dlopen(sLib, RTLD_NOW);
-- 
2.34.1

